hms-app/src/components/HackathonDetails.tsx

```diff
@@ -116,6 +217,45 @@ export default function HackathonDetails(props: IProps) {
     }
   }, [ideaData])
 
+  const participantsPreviewData = hackathonData.participants?.map(
+    (participant, index) => (
+      <SimpleGrid
```

Comment:
same thing here (and in the other lists) about them being wrapped in `<li>` and having a `key` set